Lib/test/test_ssl.py

```diff
@@ -4434,6 +4434,35 @@ def test_pha_not_tls13(self):
                 s.write(b'PHA')
                 self.assertIn(b'WRONG_SSL_VERSION', s.recv(1024))
 
+    def test_bpo37337_pha_cert_none(self):
```

Comment:
Is `bpo37337` correct here? The issue addressed seems to be: https://bugs.python.org/issue37428 

This might be more of a question.
I thouhght, the test should have two parts.

1. Server verifying `SSL_VERIFY_POST_HANDSHAKE` flag set.
2. Client which asserts `SSL_VERIFY_POST_HANDSHAKE` not set.

I could not figure out if the test covers both these scenarios.

Reply:
I fixed the BPO number in the name of the test.

The new test verifies that ``client_context.post_handshake_auth`` does not implicitly enable server cert validation on the client side. There are already several other tests that verify several combinations of PHA.

* the test case ``test_pha_required`` verifies that ``SSL_VERIFY_POST_HANDSHAKE`` on a server context works as expected. The server does request a client during handshake.
* ``SSL_VERIFY_POST_HANDSHAKE`` must not be set on a client context. This was the original bug. The ``_set_verify_mode`` helper prevents this already.

Reply:
Thanks for answering these, @tiran. LGTM.